README.md

```
@@ -43,37 +58,40 @@ Repository with Kitchen production line resources
     - Status changes cannot be rolled back.
     - If the order status is `Completed`, it should be removed from the orders list.
 - **Events**:
-    - An `OrderStatusUpdated` event should be triggered on every status change operation.
+    - An `OrderStatusChanged` event should be triggered on every status change operation.
```

Markdownlint (0.35.0)

Expected: 2; Actual: 4
Unordered list indentation

(MD007, ul-indent)